### ISSUE TITLE
Typescript types.d.ts missing

### ISSUE THREAD

**Issue by drudunn** [adobe/aem-react-editable-components#10]
Unlike the other new [AEM](https://github.com/adobe/aem-spa-page-model-manager) [libraries](https://github.com/adobe/aem-spa-component-mapping), the `types.d.ts` for this project is missing upon installation

```
Could not find a declaration file for module '@adobe/aem-react-editable-components'. 
Try `npm install @types/adobe__aem-react-editable-components` if it exists or add a new declaration (.d.ts) file containing `declare module '@adobe/aem-react-editable-components';`ts(7016)
```

The types for this package are not present in `node_modules`. I thought to check in case they were not in a different folder to `dist`, but no.
![Screenshot-2020-08-27-at-10 45 08-am](https://user-images.githubusercontent.com/7219196/91370936-0a3e3500-e853-11ea-91f5-e04afc8c5737.png)

Example Repo with TS error — https://codesandbox.io/s/aem-react-editable-components-cx9q0?file=/src/components/ContentTitle.tsx
```
Cannot find module '@adobe/aem-react-editable-components' or its corresponding type declarations.ts(2307)
```

Note the other libraries are also included in the example repo, and do not have TS errors.

**Comment by josephrignanese** [comment]
Hi @drudunn, did you do an `npm run build`

**Comment by drudunn** [comment]
That does produce the correct types & `dist` folder in the repo locally yes.

The issue is that they are not present when installing the package via npm/yarn.

**Comment by niekraaijmakers** [comment]
I'm pretty confident this is because there is no .npmignore file.

That means that it falls back to .gitignore and doesn't include those files.
The other project does have one:

https://github.com/adobe/aem-spa-component-mapping/blob/development/.npmignore
https://github.com/adobe/aem-spa-page-model-manager/blob/development/.npmignore

If we introduce this and republish, it should be fixed

**Comment by CezCz** [comment]
Thanks guys, released v1.0.1 @niekraaijmakers solution seems to have worked.
